torch/nn/_intrinsic/qat/modules/conv_fused.py

```diff
@@ -80,13 +80,13 @@ def reset_parameters(self):
             self.reset_bn_parameters()
 
     def enable_fake_quant(self):
-        self.observer.enable()
-        self.weight_fake_quant.enable()
+        self.observer.enable_fake_quant()
```

Comment:
I feel the ordering of the PRs is weird as you delete it later :) but it's ok

Reply:
Yes, on hind-sight having fewer PRs would have been better.